R/utils-metrics.R

```diff
@@ -0,0 +1,59 @@
+#' Get a data frame with 1 row for each metric, matching the metric with the
+#' output type to use for calculating the metric.  If the metric is invalid or
+#' can't be calculated from the available output types for the target, the
+#' output_type will be NA.
+#'
+#' This implementation is somewhat fragile.  It assumes that all metrics are
+#' either an interval coverage (to be computed based on quantile forecasts) or
+#' a standard metric provided by scoringutils.  If hubEvals eventually supports
+#' other metrics, this function will need to be updated.
+#'
+#' @noRd
+get_metric_name_to_output_type <- function(task_groups_w_target, metrics) {
```

Comment:
**suggestion**

What you are creating here is a lookup table, so I would name the function something similar. 

```suggestion
get_metric_name_lookup_table <- function(task_groups_w_target, metrics) {
```

Reply:
I prefer to keep both the "from" and "to" variables in the name, to give more clarity about what variables are involved and what the intended direction of the lookup is.  I considered `get_lookup_metric_to_output_type`, but it is too long.  So I am sticking with the current name.